src/main/kotlin/traffic_simulation/roadNetwork.kt

```diff
@@ -0,0 +1,37 @@
+package traffic_simulation
+
+class roadNetwork(val capacity : Double){
+
+    //methods use lists of vehicles; the lists origin is not specified at the moment
+    //as the functions dont need to know the origin to work
+
+    fun calculateDemand(vehicles:MutableList<vehicle>): Double{
```

Comment:
i will consider this points and write my next commits in german again... guess its getting late